vol_file_test.c

```diff
@@ -3151,6 +3245,8 @@ cleanup_files(void)
     HDremove(FILE_INFO_FILENAME);
     HDremove(DOUBLE_GROUP_OPEN_FILENAME);
 #endif /* for native VOL connector test only */
+
+error:
```

Comment:
Ah that's a bit awkward needing the `error:` label in the cleanup section now due to the use of the TEST_ERROR macro, especially since cleanup_files doesn't return anything. Maybe we should have a small utility routine in vol_test_util.c that just prefixes the filename, calls H5Fdelete on the file and then frees the prefix buffer so that we can have the `error:` label there instead and just call the utility routine here. In fact, it would probably be a good idea to also have a utility routine to encompass prefixing the filename, calling H5Fcreate/H5Fopen, freeing the prefix buffer and then returning the `hid_t` for the file, but we can always do that later in the interest of getting this PR merged.